locales/en/app.ftl

```diff
@@ -210,10 +210,11 @@ unsub-headline = Unsubscribe from {-product-name-nowrap}
 unsub-blurb = This will remove your email from the {-product-name-nowrap} list and you will no longer receive alerts when new breaches are announced. 
 unsub-button = Unsubscribe
 
-unsub-survey-headline = You are no longer subscribed.
-unsub-survey-blurb = 
-  Your email is unsubscribed from {-product-name-nowrap}. Thank you for using this service. 
-  Will you take a moment to answer one question about your experience?
+fxa-unsub-headline = Unsubscribe from {-product-name} alerts.
+fxa-unsub-blurb = You’ll no longer receive {-product-name} alerts. 
+  Your {-brand-fxa} will remain active, and you may receive other 
+  account-related communications.
```

Comment:
Nit: should we include a link to manage their FxA settings if they want to nuke that too?

Reply:
Yep, the link from the above comment follows this string.